src/types/__test__/system.test.ts

```diff
@@ -196,3 +216,98 @@ describe.skip('Search', () => {
+
+    const features = (data?.official?.features || []).reduce(reducer, {})
+
+    expect(features.add_credit).toBe(true)
+    expect(features.circle_management).toBe(true)
+    expect(features.circle_interact).toBe(true)
+    expect(features.fingerprint).toBe(true)
+    expect(features.payment).toBe(true)
+    expect(features.payout).toBe(true)
+    expect(features.verify_appreciate).toBe(false)
+  })
+
+  test('update feature flag', async () => {
+    const { query, mutate } = await testClient(userClient)
+
+    const updateData = await mutate({
+      mutation: SET_FEATURE,
+      variables: { input: { name: 'circle_management', flag: 'off' } },
+    })
+    expect(_get(updateData, errorPath)).toBe('FORBIDDEN')
+
+    // set feature off
+    const { query: adminQuery, mutate: adminMutate } = await testClient(adminClient)
+    const updateData2 = await adminMutate({
+      mutation: SET_FEATURE,
+      variables: { input: { name: 'circle_management', flag: 'off' } },
+    })
+    expect(_get(updateData2, 'data.setFeature.enabled')).toBe(false)
+
+    const { data: queryData } = await query({ query: QUERY_FEATURES })
+    const features = (queryData?.official?.features || []).reduce(reducer, {})
+    expect(features.circle_management).toBe(false)
+
+    const { data: queryData2 } = await adminQuery({ query: QUERY_FEATURES })
+    const features2 = (queryData2?.official?.features || []).reduce(reducer, {})
+    expect(features2.circle_management).toBe(false)
+
+    // set feature as admin
+    const updateData3 = await adminMutate({
+      mutation: SET_FEATURE,
+      variables: { input: { name: 'circle_management', flag: 'admin' } },
+    })
+    expect(_get(updateData3, 'data.setFeature.enabled')).toBe(true)
+
+    const { data: queryData3 } = await query({ query: QUERY_FEATURES })
+    const features3 = (queryData3?.official?.features || []).reduce(reducer, {})
+    expect(features3.circle_management).toBe(false)
+
+    const { data: queryData4 } = await adminQuery({ query: QUERY_FEATURES })
+    const features4 = (queryData4?.official?.features || []).reduce(reducer, {})
+    expect(features4.circle_management).toBe(true)
+
+    // set feature as seeding
+    const updateData4 = await adminMutate({
+      mutation: SET_FEATURE,
+      variables: { input: { name: 'circle_management', flag: 'seeding' } },
+    })
+    expect(_get(updateData4, 'data.setFeature.enabled')).toBe(true)
+
+    const { data: queryData5 } = await query({ query: QUERY_FEATURES })
+    const features5 = (queryData5?.official?.features || []).reduce(reducer, {})
+    expect(features5.circle_management).toBe(false)
+
+    const { data: queryData6 } = await adminQuery({ query: QUERY_FEATURES })
+    const features6 = (queryData6?.official?.features || []).reduce(reducer, {})
+    expect(features6.circle_management).toBe(true)
+
+    const { query: otherQuery } = await testClient({ isAuth: true, isOnboarding: true })
```

Comment:
Are we missing a mutation to toggle users in seeding list?

Reply:
I'll put it on soon.